dts/arm/nuvoton/npcx.dtsi

```diff
@@ -691,6 +691,13 @@
 			label = "I2C_7_PORT_0";
 			status = "disabled";
 		};
+
+		power_domain_psl: power-domain-psl {
+			compatible = "nuvoton,npcx-power-domain-psl";
+			label = "POWER_DOMAIN_PSL";
+			enable-gpios = <&gpio8 5 0>;
```

Comment:
I'm a bit confused by this, is it used as an output and it has to be GPIO85 because it's internally connected to PSL_OUT (trying to understand why this is in not in the board dts)? Then it would probably need flags as well.

Reply:
> I'm a bit confused by this, is it used as an output and it has to be GPIO85 because it's internally connected to PSL_OUT (trying to understand why this is in not in the board dts)? 

Yes, it is connected to GPIO85 internally. I agree with you that it has a better understanding how PSL_OUT works if we move this property in the board dts files. Will do it later.

Reply:
Done.